07-bootstrap-validation.md

```diff
@@ -56,18 +56,19 @@ GitOps allows a team to author Kubernetes manifest files, persist them in their
 > Mindful Timing for Upgrades:
 > - Be mindful of when upgrades should occur. If you have overlapping maintenance windows, AKS will determine the running order.
 > - To avoid conflicts, leave at least 24 hours between maintenance window configurations. The timing will depend on the number of nodes in your specific cluster and the duration required for upgrades.
+> - Current configuration should not represent a conflict since Kubernetes version updates are applied manually when customer see fit.
 >
 > OS-Level Updates:
-> - By default, the OS-level updates maintenance window is scheduled on a weekly cadence. This is because the OS channel is configured with `NodeImage`, where a new node image is shipped every week.
-> - If you choose the `SecurityPatch` channel, consider changing the maintenance window to daily for more frequent updates.
+> - By default, the OS-level updates maintenance window is scheduled on a daily cadence. This is because the OS channel is configured with `SecurityPatch`, where a new update can be shipped when available.
+> - If you choose the `NodeImage` channel, consider changing the maintenance window to weekly since updates get shipped on a weekly cadence.
```

Comment:
I'd actually show the maintenance window being on Tue, Thurs, and the weekend.  Just because you COULD get security updates daily doesn't mean you will nor does it mean your maintenance window MUST align with that possibility.  It would be good to show that the cluster operator still has control here.

Reply:
@ckittel auto-upgrade channels won't allow us to configure two different days of the weeks as requested. We are now moving over weekly maintenance window following official docs recommendations for `NodeImage` channel. 

Open to keep discussing alternatives as I also think the two-days-catchingup is a great pattern.

partially done |  attempt to addressed from abe4669d37d515c3ad2e9aa4f9c2d2afe6e85c77 and  a5257c2409fcb672b1172d3b7dc2f673a83479ce